Adds a Syringe to the Uplink Poison/Chemical kit

torger597 opened the pull request:
Useful for:
1: Poisoning food.
2: Injecting Chemicals directly rather than being forced to feed someone
from a bottle.

torger597 committed:
Adds a Syringe to the Uplink Poison/Chemical kit

Useful for:
1: Poisoning food.
2: Injecting Chemicals directly rather than being forced to feed someone
from a bottle.

Iamgoofball commented:
:+1:

KorPhaeron commented:
Should change the poison kit desc (which says it doesnt come with any delivery method) and maybe changelog it

torger597 commented:
Sorry, will do. Thanks for reminding me.

torger597 committed:
Edit

Changed description of the kit itself to comment on the fact it now
comes with a syringe, added a change log entry for the change itself.
